apps/nowcasting-app/components/globalState.tsx

```diff
@@ -12,14 +13,29 @@ export function get30MinNow() {
   }
   return date.toISOString();
 }
+
+export function getFloor30MinNow() {
```

Comment:
you are not using this hook, can we delete it if it's not necessary please

Reply:
thanks